### ISSUE TITLE
Error code 400 when testing a action

### ISSUE THREAD

**Issue by dinhienhy** [frapi/frapi#112]
Hi all,

I have installed FRAPI on Win 7 with apache web server.
When I try to test a action, I have recieved bellow result:

Response (Code 400):
Date: Sat, 10 Mar 2012 01:51:40 GMT
Server: Apache/2.2.17 (Win32) mod_ssl/2.2.17
    OpenSSL/0.9.8o PHP/5.3.4 mod_perl/2.0.4
    Perl/v5.10.1
X-powered-by: PHP/5.3.5
Access-control-allow-origin: *
Access-control-allow-headers: *
Access-control-allow-credentials: true
Access-control-allow-methods: POST,GET,PUT,DELETE,HEAD
Access-control-max-age: 604800
Content-length: 503
Connection: close
Content-type: application/xml; charset=utf-8

Content Body:
{"errors":[{"message":"Argument 1 passed to Frapi_Action::setAcceptParams() must be an array, boolean given, called in C:\frapi\src\frapi\library\Frapi\Controller\Api.php on line 245 and defined (Error Number: 4096), (File: C:\frapi\src\frapi\library\Frapi\Action.php at line 160)","name":"PHP Recoverable error","at":""}]}

This action has no parameter.
Let me know if I have any mistake?

Thanks,
Hien Phan

**Comment by dshafik** [comment]
This has already been fixed in PR #105 — it just needs to be merged in. In the meantime if you want to pull that change in you can do:

``` bash
$ git remote add dshafik git://github.com/dshafik/frapi.git
$ git fetch
$ git checkout dshafik/mimetype-params-versioning
```

If you do give that a try, it would be great if you can confirm it works on the PR and I'll go ahead and merge it to master.

**Comment by dinhienhy** [comment]
Frapi has worked.
Thanks dshafik,
